### PR TITLE
Fix formatting so functions link

### PR DESCRIPTION
**Pull request by kaitejohnson**
Edited:
The reason things weren't linking is just that they weren't formatted properly. Link citations only necessary if you have a bibliograph (which we don't)

### CODE REVIEW

**Comment by dylanhmorris** [comment]
This appears to work for me. Only issue I see in the rendered version is one instance of `roxygen2`-style brackets, which can be removed.

https://github.com/CDCgov/ww-inference-model/blob/b51a1e24ccd7e821ce0e12d9944d54eb21b16981/vignettes/wwinference.Rmd#L419

**Comment by kaitejohnson** [comment]
Weird ok I will address both in this PR. @dylanhmorris Does it work when you render locally or just when you preview the package down site? (Which I have tried, and somehow still have not figured out but need to try harder..)

**Comment by gvegayon** [comment]
I don't see any citations in the vignette; what are you using that option for, @kaitejohnson?

**Comment by kaitejohnson** [comment]
> I don't see any citations in the vignette; what are you using that option for, @kaitejohnson?

Despite the name, I think this is what makes `function()` link to the function documentation in the vignette.

File: vignettes/wwinference.Rmd [hunk 1]
```diff
@@ -162,7 +162,7 @@ below the LOD in upstream pre-processing.
 
 ## Hospital admissions data pre-processing
 
-The `preprocess_hosp_data`  function standardizes the column names of the
+The `preprocess_hosp_data()`  function standardizes the column names of the
```
Comment (dylanhmorris) [inline vignettes/wwinference.Rmd:165]: Update function name
```suggestion
The `preprocess_count_data()`  function standardizes the column names of the
```

File: vignettes/wwinference.Rmd
Comment (dylanhmorris) [inline vignettes/wwinference.Rmd:1]: do you want to refer to the function `wwinference()` in line 62?

Comment (dylanhmorris) [reply vignettes/wwinference.Rmd:1]: Not for this PR but the "model specification" section is not rendering properly.

Comment (dylanhmorris) [reply vignettes/wwinference.Rmd:1]: l. 329 `model_spec()` should be `get_model_spec()`

Comment (dylanhmorris) [reply vignettes/wwinference.Rmd:1]: link to bayesplot tutorial is backwards and therefore not rendering.

written as 
```markdown
we recommend the (bayesplot tutorial)[https://mc-stan.org/bayesplot/articles/visual-mcmc-diagnostics.html].
```

Should be:

```markdown
we recommend the [bayesplot tutorial](https://mc-stan.org/bayesplot/articles/visual-mcmc-diagnostics.html).

```

**Review by dylanhmorris — COMMENTED** [review]
Links work for me, but found some formatting things and typos while reviewing.

File: vignettes/wwinference.Rmd
Comment (kaitejohnson) [reply vignettes/wwinference.Rmd:1]: It should be `wwinference` referring to the package. Will clarify.

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**